src/__test__/components/plots-and-tables/ViolinPlotIndex.test.jsx

```diff
@@ -124,6 +124,7 @@ describe('ViolinIndex', () => {
   });
 
   it('allows selection of the grouping/points, defaulting to louvain and All', async () => {
+    jest.setTimeout(30000);
```

Comment:
what was the reason for adding this? Did you observe your unit tests failing because of it?

Reply:
I am asking because I am experiencing some intermittent problems with the unit tests in this file (@xverges knows about it) and I wonder if you observed it too.

Reply:
This needs to be removed. I think that the tests will pass consistently once #237 is merged

Reply:
we didn't manage to fix the tests in that PR. Stefan, can you confirm the reason for adding this line?

Reply:
it did fix it, it was failing because the timeout was expiring, but its good now I'll remove it.

Reply:
the tests are still not fixed, but we will fix them as part of later work. Sounds good for now